test/resourceBase.js

```diff
@@ -33,7 +33,7 @@ describe('resource base', () => {
       }
     });
 
-    return resource._transmit('POST', null, null, null, (err) => {
+    resource._transmit('POST', null, null, null, (err) => {
```

Comment:
why did you have to make this change?

Reply:
Here we're returning a promise AND defining a callback function in the `_transmit` call; newer version of mocha don't let you do both. Here's what the error looks like when we do:

```
 1) resource base
       should expose the raw response on 500 level errors:
     Error: Resolution method is overspecified. Specify a callback *or* return a Promise; not both.
      at resource._transmit (test/resourceBase.js:38:14)
      at tryCatcher (node_modules/bluebird/js/release/util.js:16:23)
      at Promise.errorAdapter [as _rejectionHandler0] (node_modules/bluebird/js/release/nodeify.js:35:34)
      at Promise._settlePromise (node_modules/bluebird/js/release/promise.js:566:21)
      at Promise._settlePromise0 (node_modules/bluebird/js/release/promise.js:614:10)
      at Promise._settlePromises (node_modules/bluebird/js/release/promise.js:689:18)
      at Async._drainQueue (node_modules/bluebird/js/release/async.js:133:16)
      at Async._drainQueues (node_modules/bluebird/js/release/async.js:143:10)
      at Immediate.Async.drainQueues (node_modules/bluebird/js/release/async.js:17:14)
```